### PR TITLE
Adding local roles on @search scope

### PR DESCRIPTION
**Pull request by bloodbare**

### CODE REVIEW

**Review by masipcat — APPROVED** [review]

**Comment by bloodbare** [comment]
@masipcat you should also apply it to guillotina elasticsearch, as it does not check local roles of the user

File: guillotina/contrib/catalog/pg.py [hunk 1]
```diff
@@ -487,6 +488,7 @@ def get_default_where_clauses(self, container: IContainer) -> typing.List[str]:
         users.extend(principal.groups)
         roles_dict = policy.global_principal_roles(principal.id, principal.groups)
         roles.extend([key for key, value in roles_dict.items() if value])
+        roles = get_roles_principal(container)
```
Comment (vangheem) [inline guillotina/contrib/catalog/pg.py:491]: You are overriding the `roles` value used above. Is that on purpose?

Comment (vangheem) [reply guillotina/contrib/catalog/pg.py:491]: Should it be an extension?

Comment (bloodbare) [reply guillotina/contrib/catalog/pg.py:491]: ops I forgot to delete it, get_roles_principal already gets the global ones, so its not an extend

Comment (vangheem) [reply guillotina/contrib/catalog/pg.py:491]: ahh right, that makes sense.

Comment (bloodbare) [reply guillotina/contrib/catalog/pg.py:491]: Fixed at https://github.com/plone/guillotina/commit/bd1468066189c8fe95b52a1f0002070b54aaeafd (sorry I did not do a PR)

**Pull request merged**